src/main/resources/application.yml

```diff
@@ -0,0 +1,5 @@
+spring:
```

Comment:
new yaml file is missing this spring.devtools.remote.secret=mysecret.
Can you add that too? It is needed for livereload